### ISSUE TITLE
When using with Cron syntax.

### ISSUE THREAD

**Issue by wasabeef** [peter-evans/create-pull-request#29]
Thank you for the great plugin.

When I using with Cron schedule syntax.

```
on:
 schedule:
 - cron: '*/5 * * * *'
```

That happened this error.
```
Traceback (most recent call last):
  File "/create-pull-request.py", line 134, in <module>
    if not ignore_event(github_event):
  File "/create-pull-request.py", line 33, in ignore_event
    deleted = "{deleted}".format(**github_event)
KeyError: 'deleted'
##[error]Docker run failed with exit code 1
```

**Comment by vitorarantes** [comment]
same here when using schedule event:

```
Traceback (most recent call last):
  File "/create-pull-request.py", line 119, in <module>
     if not ignore_event(github_event):
  File "/create-pull-request.py", line 23, in ignore_event
    deleted = "{deleted}".format(**github_event)
KeyError: 'deleted'
##[error]Docker run failed with exit code 1
```

**Comment by stefanbuck** [comment]
The issue seems to be related to the fact, that on scheduled jobs the `$GITHUB_EVENT_PATH` file contains just one attribute 

```json
{
  "schedule": "* * * * *"
}
```

whereas on `push` it contains a proper GitHub payload with `repository` and `sender` details.


I'm not really experienced with Python to come up with a bug fix PR, but might this helps someone else to fix the issue. From what I can tell it's this [ignore_event function](https://github.com/peter-evans/create-pull-request/blob/facb42d77609ff5c9b181a506486ee31c2a425f8/create-pull-request.py#L17-L31) which needs a `deleted` along with a `ref` key in that json.

**Comment by peter-evans** [comment]
Apologies for the slow response to this issue. I will take a look asap.

**Comment by peter-evans** [comment]
Released version 1.1.2 to fix this. Thank you @stefanbuck for the contribution!
Please try `uses: peter-evans/create-pull-request@v1.1.2`

**Comment by peter-evans** [comment]
I had chance to properly test this today and noticed a bug with `v1.1.2`. I've released a new version and tested it. Please use `peter-evans/create-pull-request@v1.1.3`

Closing this issue. Please reopen if necessary.
